marlowe-runtime/test/Language/Marlowe/Runtime/Transaction/ConstraintsSpec.hs

```diff
@@ -100,7 +102,9 @@ spec :: Spec
 spec = do
   describe "solveInitialTxBodyContent" do
     prop "satisfies the constraints" \(SomeTxConstraints marloweVersion constraints) -> do
-      protocol <- hedgehog genProtocolParameters
+      protocol <-
+        fromRight undefined . convertToLedgerProtocolParameters ShelleyBasedEraBabbage
```

Comment:
How about `either (error . show) id` instead of `fromRight undefined`? Also, we might also want to grep for other uses of `undefined`.